test.py

```diff
@@ -22,50 +15,51 @@ class TestFitness(unittest.TestCase):
     def test_onemax():
         """Test OneMax fitness function"""
         state = np.array([0, 1, 0, 1, 1, 1, 1])
-        assert(OneMax().evaluate(state) == 5)
+        assert OneMax().evaluate(state) == 5
 
     @staticmethod
     def test_flipflop():
         """Test FlipFlop fitness function"""
         state = np.array([0, 1, 0, 1, 1, 1, 1])
-        assert(FlipFlop().evaluate(state) == 3)
+        assert FlipFlop().evaluate(state) == 3
 
     @staticmethod
     def test_head():
         """Test head function"""
         state = np.array([1, 1, 1, 1, 0, 1, 0, 2, 1, 1, 1, 1, 1, 4, 6, 1, 1])
-        assert(head(1, state) == 4)
+        assert head(1, state) == 4
 
     @staticmethod
     def test_tail():
         """Test tail function"""
         state = np.array([1, 1, 1, 1, 0, 1, 0, 2, 1, 1, 1, 1, 1, 4, 6, 1, 1])
-        assert(tail(1, state) == 2)
+        assert tail(1, state) == 2
 
     @staticmethod
     def test_max_run():
         """Test max_run function"""
         state = np.array([1, 1, 1, 1, 0, 1, 0, 2, 1, 1, 1, 1, 1, 4, 6, 1, 1])
-        assert(max_run(1, state) == 5)
+        assert max_run(1, state) == 5
 
     @staticmethod
     def test_queens():
         """Test Queens fitness function"""
         state = np.array([1, 4, 1, 3, 5, 5, 2, 7])
-        assert(Queens().evaluate(state) == 6)
+        assert Queens().evaluate(state) == 6
 
-
-def test_max_k_color(self):
+    @staticmethod
```

Comment:
Is it necessary to have the `@staticmethod` decorators? They aren't used in the example included in the `unittest` documentation: [https://docs.python.org/3/library/unittest.html](url)

Reply:
OK. Thanks for explaining that. I haven't used @staticmethod before in Python, and don't have much experience with Java, so was uncomfortable with their use.